docs/debugging-eventlisteners.md

```diff
@@ -1,43 +1,50 @@
 <!--
 ---
-linkTitle: "Debugging EventListeners"
+linkTitle: "Troubleshooting"
```

Comment:
Should this be called Troubleshooting Eventlisteners instead?

Also, since we changed the title, maybe we should also change the name of the file to match?

Reply:
I think we should rename this file to troubleshooting.md file in a separate PR in which we also update all links currently pointing to debugging-eventlisteners.md to point to troubleshooting.md. The goal is to have a Troubleshooting page instead of just a Debugging EventListeners page.